CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Yii Queue Change Log
+# Queue Change Log
```

Comment:
```suggestion
# Yii Queue Change Log
```

We change package name to `yiisoft/queue` only, but pretty name keep as `Yii Queue`